### PR TITLE
Remove starkEx version flag

### PR DESCRIPTION
**Pull request by plutoegg**
starkExV2 is now the default (and only) version

### CODE REVIEW

**Comment by Matko95** [comment]
The flag is also used on some other places as well (ledger/authereum flows..)

**Comment by plutoegg** [comment]
> The flag is also used on some other places as well (ledger/authereum flows..)

Hmm i dont find anything in this repo, can you point me at them?

**Comment by Matko95** [comment]
@plutoegg
https://github.com/DeversiFi/dvf-client-js/blob/590b55e90243ab208c5be42dd5a5e5b58872d5e7/src/lib/ledger/provideContractData.js#L26

https://github.com/DeversiFi/dvf-client-js/blob/590b55e90243ab208c5be42dd5a5e5b58872d5e7/src/lib/stark/ledger/createSignedOrder.js#L86

https://github.com/DeversiFi/dvf-client-js/blob/590b55e90243ab208c5be42dd5a5e5b58872d5e7/src/lib/stark/ledger/createSignedTransfer.js#L30

**Comment by plutoegg** [comment]
Fixed, but couldnt find anything related to Authereum

File: src/api/contract/getStarkKey.js [hunk 1]
```diff
@@ -2,22 +2,5 @@ const DVFError = require('../../lib/dvf/DVFError')
...
-      'getStarkKey',
-      args
-    )
-
-    return new BN(starkKeyDecimal).toString(16)
-  } catch (e) {
-    console.log('contract/getStarkKey error is: ', e)
-    throw new DVFError('ERR_GETTING_STARK_KEY')
-  }
+  throw new Error('getStarkKey contract method has been removed in StarkExV2}')
```
Comment (adrian-gierakowski) [inline src/api/contract/getStarkKey.js:5]:
```suggestion
  throw new Error('getStarkKey contract method has been removed in StarkExV2')
```

**Review by adrian-gierakowski — APPROVED** [review]
looks good

**Pull request merged**